### PR TITLE
Olasupov patch 1

### PR DESCRIPTION
**Pull request by olasupov**
Please check my patch pull request for Bugzilla Bug 1202071 for changing the email template sent for an add-on reviews.

### CODE REVIEW

File: apps/editors/templates/editors/emails/nominated_to_nominated.ltxt [hunk 1]
```diff
@@ -1,3 +1,8 @@
+my-addon/
+         data
+         lib
+         locale/
+                en-US.properties
```
Comment (wagnerand) [inline apps/editors/templates/editors/emails/nominated_to_nominated.ltxt]: What are those for? They seem unrelated.

**Comment by wagnerand** [comment]
Can you squash all commits into one, please?

Please do not open a new pull request each time you update your commit(s).

**Comment by wagnerand** [comment]
@olasupov are you still working on this?

**Comment by olasupov** [comment]
Hi, not anymore unfortunately.
